Double quotes in passwords are not escaped

First of all, thank you for creating this wonderful piece of kit! It is ridiculously useful.

We're using SASL-SCRAM login on our Kafka, but looking at the code this issue would be applicable to all JAAS usernames and passwords.
Our passwords are randomly generated and occasionally contain double quote characters (`"`). The config for the gitops user looks somewhat like this:

```
sasl.jaas.config=org.apache.kafka.common.security.scram.ScramLoginModule required username="my-gitops-user" password="XXXXXXXXX"YYYYY";
```

Then when running 'kafka-gitops plan` it gives this error:
```
[ERROR] Error thrown when creating Kafka admin client:
Value not specified for key 'YYYYY' in JAAS config
```

I had a look at the code. And it looks easy enough to fix. I'll submit a PR for this later tonight if that's ok with you.

--- a/src/main/java/com/devshawn/kafka/gitops/config/KafkaGitopsConfigLoader.java
+++ b/src/main/java/com/devshawn/kafka/gitops/config/KafkaGitopsConfigLoader.java
@@ -68,12 +68,19 @@ public class KafkaGitopsConfigLoader {
             }
 
             String value = String.format("%s required username=\"%s\" password=\"%s\";",
-                    loginModule, username.get(), password.get());
+                    loginModule, escape(username.get()), escape(password.get()));
             config.put(SaslConfigs.SASL_JAAS_CONFIG, value);
         } else if (username.get() != null) {
             throw new MissingConfigurationException("KAFKA_SASL_JAAS_PASSWORD");
         } else if (password.get() != null) {
             throw new MissingConfigurationException("KAFKA_SASL_JAAS_USERNAME");
         }
+    }
+
+    private static String escape(String value) {
+        if (value != null) {
+            return value.replace("\"", "\\\"");
+        }
+        return null;
     }
 }

